packages/amplication-client/src/Plugins/hooks/usePlugins.ts

```diff
@@ -1,16 +1,22 @@
-import { useMutation, useQuery } from "@apollo/client";
+import { QueryOptions, useMutation, useQuery } from "@apollo/client";
 import {
   GET_PLUGIN_INSTALLATIONS,
   CREATE_PLUGIN_INSTALLATION,
   UPDATE_PLUGIN_INSTALLATION,
   GET_PLUGIN_ORDER,
   UPDATE_PLUGIN_ORDER,
+  GET_PLUGIN_INSTALLATION,
 } from "../queries/pluginsQueries";
 import * as models from "../../models";
 import { keyBy } from "lodash";
 import { useCallback, useContext, useEffect, useMemo, useState } from "react";
 import { AppContext } from "../../context/appContext";
 
+export type PluginVersion = {
```

Comment:
please takeout all types to separate file

Reply:
this will be inmported from the Plugin API soon - we can leave it here instead of doing it twice